src/main/java/com/prgrms/commandLineApplication/io/Console.java

```diff
@@ -73,14 +65,14 @@ public void printMenuError() {
   @Override
   public void printAllVoucher(List<Voucher> vouchers) {
     for (Voucher voucher : vouchers) {
-      DiscountType discountType = voucher.getDiscount().getDiscountType();
-      System.out.println(String.format("- %s : %s", UUID, voucher.getVoucherId()));
-      System.out.println(String.format("- %s %s : %s%s", discountType, VOUCHER, voucher.supplyDiscountAmount(), printUnit(discountType)));
+      DiscountType discountType = voucher.supplyDiscountType();
+      System.out.println(String.format("- UUID : %s", voucher.getVoucherId()));
+      System.out.println(String.format("- %s VOUCHER : %s%s", discountType, voucher.supplyDiscountAmount(), printUnit(discountType)));
     }
   }
 
   private String printUnit(DiscountType discountType) {
-    return FIXED.equals(discountType.name()) ? UNIT_WON : UNIT_PERCENT;
+    return "FIXED".equals(discountType.name()) ? "₩" : "%";
```

Comment:
```suggestion
    return DiscountType.FIXED == discountType ? "₩" : "%"
```
그리고 String 보다는 DiscountType을 사용해서 이렇게 비교하는 것이 좋아보이네요

Reply:
803a76b5c8af7f66592ec7cfaa16f95a33116dae